src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -106,16 +130,41 @@ private void validateCeasedDate(OverseasEntitySubmissionDto overseasEntitySubmis
             if (UtilsValidators.isNotNull(trustCeasedDate, qualifiedFieldName, errors, loggingContext)) {
                 DateValidators.isDateInPast(trustCeasedDate, qualifiedFieldName, errors, loggingContext);
                 DateValidators.isCeasedDateOnOrAfterCreationDate(trustCeasedDate, creationDate, qualifiedFieldName, errors, loggingContext);
-            }
+
+                if (trustDataDto.getIndividuals() != null && !trustDataDto.getIndividuals().isEmpty()) {
+                    DateValidators.isCeasedDateOnOrAfterIndividualsDateOfBirth(trustCeasedDate, trustDataDto.getIndividuals(), qualifiedFieldName, errors, loggingContext);
```

Comment:
not sure why we need to check the trust ceased date against the individuals date of birth, it should be that the trust ceased date is after the trust start date. Check that one with Vernon.

Reply:
See the web ticket linked to this.

Reply:
UAR-1511 relates to https://companieshouse.atlassian.net/browse/UAR-1449, which is the 'review the trust page'. I think there may have been a mix up, as [UAR-1516](https://companieshouse.atlassian.net/browse/UAR-1516) is for the individuals ceased date and [UAR-1517](https://companieshouse.atlassian.net/browse/UAR-1517) is for the legal entities ceased date.

Reply:
I agree. UAR-1499 is the corresponding web ticket. The other two 'blocks' tickets are not relevant to this story.

Reply:
I also agree and was never disputing which ticket it was, it's just on UAR-1499 there was mention of dates of birth I see it has now been removed from the ticket so will remove from the PR.

Reply:
Yes, I spotted the DOB clause on that ticket yesterday. Questioned it with Vern and he then removed it as it was wrong.

Reply:
Removed this.